gradle/libs.versions.toml

```diff
@@ -0,0 +1,21 @@
+[versions]
```

Comment:
Why use toml file? The downside I see, the autocompletion in IDE is missing.

Reply:
Chaning something in TOML files doesn't cause recompilation of Gradle scripts, and it's a standard way to gather dependencies in a single place. I prefer it over having versions somewhere in the code